acceptance/server_test.go

```diff
@@ -116,12 +221,9 @@ func AddHandlers(server *TestServer) {
 	})
 
 	server.Handle("GET /api/2.0/workspace/get-status", func(r *http.Request) (any, error) {
-		return workspace.ObjectInfo{
-			ObjectId:   1001,
-			ObjectType: "DIRECTORY",
-			Path:       "",
-			ResourceId: "1001",
-		}, nil
+		path := r.URL.Query().Get("path")
+
+		return server.getStatus(path)
```

Comment:
why extra indirection with getStatus function? I don't see that helper used anywhere else. 

Same comment for workspaceMkdirs, jobsCreate, jobsList - those are only used once, can be inlined.

Reply:
Can we have a separate PR for jobs API in server_test.go (excluding changes to python mutator and tests for python mutator)?